### PR TITLE
Update README for version 2.0.0

### PR DESCRIPTION
**Pull request by jespernpedersen**
Will be merged for release.

### CODE REVIEW

**Review by Flavio-Almeida — APPROVED** [review]

File: siteimprove/readme.txt [hunk 1]
```diff
@@ -28,37 +28,41 @@ Upload the Siteimprove plugin zip file using Wordpress’ ‘Upload Plugin’ fe
...
 
-Click on ‘Login’ and enter your Siteimprove username and password to access the features.  
+Click on ‘Login’ and enter your Siteimprove username and password to access Siteimprove.  
 
 A new Siteimprove menu option will also appear on the left menu bar. Click on this menu option to continue configuring the plugin once you’ve logged in.  
 
 The token field is automatically filled and should not be changed unless a new token is required. In such cases, please click on “Request new token” to generate a new token. 
 
-API Username (Email) and API Key are API credentials that should be filled in if you have purchased the ‘Prepublish’ add-on feature. When provided correctly, you should be able to begin utilizing Prepublish.  
+Public URL can be optionally filled in, if for any reason, your published pages are not on the same URL as your configuration panel.
```
Comment (alazsiteimprove) [inline siteimprove/readme.txt:39]: After Review from Product Marketing: 
> Public URL is optional to fill in if, for any reason, your published pages are not on the same URL as your configuration panel.

File: siteimprove/readme.txt [hunk 2]
```diff
@@ -28,37 +28,41 @@ Upload the Siteimprove plugin zip file using Wordpress’ ‘Upload Plugin’ fe
...
+Click on ‘Login’ and enter your Siteimprove username and password to access Siteimprove.  
 
 A new Siteimprove menu option will also appear on the left menu bar. Click on this menu option to continue configuring the plugin once you’ve logged in.  
 
 The token field is automatically filled and should not be changed unless a new token is required. In such cases, please click on “Request new token” to generate a new token. 
 
-API Username (Email) and API Key are API credentials that should be filled in if you have purchased the ‘Prepublish’ add-on feature. When provided correctly, you should be able to begin utilizing Prepublish.  
+Public URL can be optionally filled in, if for any reason, your published pages are not on the same URL as your configuration panel. 
+
+When you download the plugin, you will be automatically be using the latest experience. If for some reason, you would like to use previous look of the Siteimprove  plugin, uncheck "Use latest experience" in the configuration panel.
```
Comment (alazsiteimprove) [inline siteimprove/readme.txt:41]: After review from Product Marketing:
> When you download the plugin, you automatically will be using the latest experience. If for some reason, you want to use the previous experience of the Siteimprove plugin, uncheck "Use latest experience" in the configuration panel.

File: siteimprove/readme.txt [hunk 3]
```diff
@@ -28,37 +28,41 @@ Upload the Siteimprove plugin zip file using Wordpress’ ‘Upload Plugin’ fe
...
+
+API Username (Email) and API Key are API credentials that should be filled in if you have purchased the ‘Prepublish’ add-on feature. When provided correctly, you should be able to begin utilizing Prepublish when you are previewing a page. 
 
 Click on ‘Save’ once you have filled out the relevant fields.  
 
 
 = Usage =
 
-Once you have completed configuring the Siteimprove plugin, clicking on the Siteimprove icon on the right sidebar will open the Siteimprove plugin, which will provide you with a full overview of your global site status and your current Digital Certainty Index (DCI) score.  
+Once you have completed configuring the Siteimprove plugin, if you are at a piece of content in the configuration panel, clicking on the Siteimprove icon on the right sidebar will open the Siteimprove plugin, which will provide you with the content's current issue and the page's current Digital Certainty Index (DCI) score.
```
Comment (alazsiteimprove) [inline siteimprove/readme.txt:50]: After review from Product Marketing:
> Once you have completed configuring the Siteimprove plugin and you are on a content page in the configuration panel, clicking on the Siteimprove icon in the right
sidebar will open the Siteimprove plugin, which will provide you with the content's current issues and the page's current Digital Certainty Index (DCI) score.

File: siteimprove/readme.txt [hunk 4]
```diff
@@ -28,37 +28,41 @@ Upload the Siteimprove plugin zip file using Wordpress’ ‘Upload Plugin’ fe
...
 
 = Usage =
 
-Once you have completed configuring the Siteimprove plugin, clicking on the Siteimprove icon on the right sidebar will open the Siteimprove plugin, which will provide you with a full overview of your global site status and your current Digital Certainty Index (DCI) score.  
+Once you have completed configuring the Siteimprove plugin, if you are at a piece of content in the configuration panel, clicking on the Siteimprove icon on the right sidebar will open the Siteimprove plugin, which will provide you with the content's current issue and the page's current Digital Certainty Index (DCI) score.  
 
-You will also be able to review QA errors across pages in your website in the ‘Site overview’ tab.  
+You will also be able to review accessibility, content and policy issues on the page you are currently editing.
 
-While editing a published page, the plugin will also provide you with a page-level DCI score alongside an assessment of the readability and quality level of the page. You can also evaluate what the impact of unpublishing the specific page may have on your overall DCI score and website quality.
```
Comment (alazsiteimprove) [inline siteimprove/readme.txt:50]: Review from Product marketing:
> While editing a published page, the plugin provides you a page-level DCI score alongside an assessment of the readability and quality level of the page. You can also evaluate what the impact of
unpublishing the specific page may have on your overall DCI score and website quality.

File: siteimprove/readme.txt [hunk 5]
```diff
@@ -28,37 +28,41 @@ Upload the Siteimprove plugin zip file using Wordpress’ ‘Upload Plugin’ fe
...
-You will also be able to review QA errors across pages in your website in the ‘Site overview’ tab.  
+You will also be able to review accessibility, content and policy issues on the page you are currently editing.
 
-While editing a published page, the plugin will also provide you with a page-level DCI score alongside an assessment of the readability and quality level of the page. You can also evaluate what the impact of unpublishing the specific page may have on your overall DCI score and website quality.  
+You can also evaluate what the impact of unpublishing the specific page may have on your overall website quality.  
 
 
 = Prepublish =
 
-If you have purchased the Prepublish add-on, you should see a Prepublish feature on the top of any unpublished page in preview mode.
```
Comment (alazsiteimprove) [inline siteimprove/readme.txt:55]: review from product marketing:
> If you have purchased the Prepublish add-on, you should see a Prepublish icon at the top of any unpublished page in preview mode.

File: siteimprove/readme.txt [hunk 6]
```diff
@@ -28,37 +28,41 @@ Upload the Siteimprove plugin zip file using Wordpress’ ‘Upload Plugin’ fe
...
+You will also be able to review accessibility, content and policy issues on the page you are currently editing.
 
-While editing a published page, the plugin will also provide you with a page-level DCI score alongside an assessment of the readability and quality level of the page. You can also evaluate what the impact of unpublishing the specific page may have on your overall DCI score and website quality.  
+You can also evaluate what the impact of unpublishing the specific page may have on your overall website quality.  
 
 
 = Prepublish =
 
-If you have purchased the Prepublish add-on, you should see a Prepublish feature on the top of any unpublished page in preview mode.  
+If you have purchased the Prepublish add-on, you should see a Prepublish feature on the top of any page in preview mode.
```
Comment (alazsiteimprove) [inline siteimprove/readme.txt:59]: review from Product marketing:
> If you have purchased the Prepublish add-on, you should see a Prepublish icon at the top of any page in preview mode.

File: siteimprove/readme.txt [hunk 7]
```diff
@@ -69,17 +73,23 @@ The plugin requires a Siteimprove subscription to be used.
 
 = Where can I see the overlay? =
 
-The overlay is visible when editing a page.
+The overlay is visible when editing a piece of content (e.g. "Pages" in WordPress configuration panel).
```
Comment (alazsiteimprove) [inline siteimprove/readme.txt:76]: review from Product marketing: 
> Did you remember to turn off your adblocker? Some adblockers do not like our iframe overlay

File: siteimprove/readme.txt [hunk 8]
```diff
@@ -69,17 +73,23 @@ The plugin requires a Siteimprove subscription to be used.
...
 
 = I don't see the overlay, whats wrong? =
 
-Did you remember to turn off your adblocker? Some adblockers does not like our iframe overlay
+Did you remember to turn off your adblocker? Some adblockers does not like our iframe overlay.
```
Comment (alazsiteimprove) [inline siteimprove/readme.txt:80]: review from product marketing:
> Did you remember to turn off your adblocker? Some adblockers do not like our iframe overlay.

**Pull request merged**